huaweicloud/config/endpoints.go

```diff
@@ -267,10 +267,6 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Name:    "gaussdb",
 		Version: "mysql/v3",
 	},
-	"gaussdbv3": {
-		Name:    "gaussdb",
-		Version: "v3",
-	},
```

Comment:
If you want to remove `gaussdbv3`, you need to remove it from the `multiCatalogKeys` map as well.

Reply:
OK